runners/console/index.ts

```diff
@@ -86,6 +86,56 @@ export class Console extends Runner {
         return this.runInstallDevonfwIde(runCommand);
     }
 
+    runRestoreWorkspace(runCommand: RunCommand): RunResult {
+        let result = new RunResult();
+        result.returnCode = 0;
+
+        let workspacesName = "workspace-" + ((runCommand.command.parameters.length > 0 && runCommand.command.parameters[0].workspace)
+            ? runCommand.command.parameters[0].workspace
+            : this.playbookName.replace("/", ""));
+
+        let workspacesDir = this.getVariable(this.useDevonCommand)
+            ? path.join(this.getWorkingDirectory(), "devonfw", "workspaces")
```

Comment:
In the other methods we use 'workspaces/main' as the workspace directory. Why now only 'workspaces'?

Reply:
Because there could be different workspaces next to each other, that you might want to load into the tutorial. You would still work mostly in the folder levels below, but that's what I understood from Jans explanation